lib/plugins.ts

```diff
@@ -1,5 +1,10 @@
 import { IGenericModule, IPlugin, IState } from './interfaces'
 
+interface ILocalStoragePluginState<S extends IState> {
+  state: S
+  __version: number
+}
```

Comment:
Because we have to check the version, we need to save both state and version in localStorage.

Reply:
I think this is a good solution 👌